packages/playground/src/components/account/account-deposit/account-deposit.tsx

```diff
@@ -30,9 +31,14 @@ export class AccountDeposit {
     this.amountDeposited = ethers.utils.parseEther(e.target.value);
 
     try {
-      await this.signer.sendTransaction({
+      const tx = {
         to: this.user.multisigAddress,
         value: this.amountDeposited
+      };
+      const gasEstimate = await this.provider.estimateGas(tx);
+      await this.signer.sendTransaction({
+        ...tx,
+        gasPrice: gasEstimate
       });
```

Comment:
Probably will look cleaner to just inline all of these vars

Reply:
@snario Doesn't seem like a blocker.